roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -30,69 +63,34 @@
     chdir: '{{ ansible_user_dir }}/workspace'
   environment: '{{ zuul | zuul_legacy_vars }}'
   when:
-    - sdk == 'gophercloud'
-    - os_branch in stable_branches
+    - '"manila" in enable_services'
```

Comment:
Expression statement is wrapped in string, seems it always be true?

Reply:
Nope, this is the string in variable list syntax required by ansible, see the reference here: https://stackoverflow.com/questions/36496911/run-an-ansible-task-only-when-the-variable-contains-a-specific-string